### PR TITLE
Implement Schema.org for Blog, BlogPost and Author component

### PR DESCRIPTION
**Pull request by code-constructor**

### CODE REVIEW

**Comment by code-constructor** [comment]
@apepper ready for review

**Comment by apepper** [comment]
`npm-debug.log` ?!?

File: src/Components/SchemaDotOrg/dataFromAuthor.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+function dataFromAuthor(author) {
+  return {
+    '@context': 'http://schema.org',
+    '@type': 'Person',
+    name: author.get('title'),
```
Comment (apepper) [inline src/Components/SchemaDotOrg/dataFromAuthor.js]: Please add `image` and `description`.

File: src/Components/SchemaDotOrg/dataFromBlog.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+function dataFromBlog(blog) {
+  return {
+    '@context': 'http://schema.org',
+    '@type': 'Blog',
+    headline: blog.get('title'),
```
Comment (apepper) [inline src/Components/SchemaDotOrg/dataFromBlog.js]: Please add `navigationBackgroundImage` as `image`.

**Pull request merged**